flink-core/src/main/java/org/apache/flink/configuration/ClusterOptions.java

```diff
@@ -131,6 +131,18 @@
                     .withDescription(
                             "Defines whether the cluster uses fine-grained resource management.");
 
+    @Documentation.Section(Documentation.Sections.EXPERT_CLUSTER)
+    public static final ConfigOption<UncaughtExceptionHandleMode> UNCAUGHT_EXCEPTION_HANDLING =
+            ConfigOptions.key("cluster.uncaught-exception-handling")
+                    .enumType(UncaughtExceptionHandleMode.class)
+                    .defaultValue(UncaughtExceptionHandleMode.FAIL)
```

Comment:
This default value will change a bit Flink's behavior. I am wondering whether this behavior change is too much or not. Maybe we should set the default to `LOG` in order to not change Flink's behavior in case of unstable libraries.